### DISCUSSION TITLE
%mem calculation in .command.trace

### DISCUSSION THREAD

**Discussion by grafitos** [nextflow-io/nextflow#2107] [Q&A] [answered]
Hello,

Is there a reason to have *10 when calculating %mem ? 

https://github.com/nextflow-io/nextflow/blob/73899f3b7967f3bd9bc393749a80ce893368a28e/modules/nextflow/src/main/resources/nextflow/executor/command-trace.txt#L33

Sometimes, this value is above 100.

Thanks !

**Comment by pditommaso** [comment, marked as answer]
To prevent floating math, later it's normalised 

https://github.com/nextflow-io/nextflow/blob/2a5cbac1e213d9d3cc962a3df1e0b89b317217d5/modules/nextflow/src/main/groovy/nextflow/trace/TraceRecord.groovy#L439

**Reply by grafitos** [reply]
Thanks !

It's because I open directly the file .command.trace (so *10). But its purpose is to be parsed afterwards. Am I right ?

**Reply by pditommaso** [reply]
yep
